app/helpers/users_helper.rb

```diff
@@ -0,0 +1,7 @@
+module UsersHelper
+  def gravatar_for user
```

Comment:
Hàm này thực hiện điều gì?

Reply:
return a Gravatar image for a given user.